resources/ui/src/ui/app/layers/customization/index.tsx

```diff
@@ -165,6 +165,25 @@ export default class Customization extends UIComponent<UI.BaseProps, UI.Customiz
     emitClient('customization.choose-gender');
   }
 
+  setInformationFirstname(e: any) {
```

Comment:
There likely wont be any need to update this one the fly. it will just be a single event when finished customizing.